src/SAT/IData.hs

```diff
@@ -61,12 +61,12 @@ import Tools.Config (Config)
   or fakes a pop action if the connection to the solver is not incremental.
```

Comment:
As I explained in my email, this interface is merely there to abstract out the kind of operations that we need to perform through the SAT solver and various packages that can make those operations available to us.